app/views/guests/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1><%= @guest.name %></h1>
+<h2>Rooms stayed in:</h2>
+<ul>
+<section id="rooms">
+  <% @rooms_stayed_in.each do |room| %>
```

Comment:
So here, following up on my earlier comment on sending only the guest object to the view, we'd just call `@guest.rooms.each ...`